configs/config.yaml.example

```diff
@@ -49,6 +49,24 @@ certs:
   cert_path: "/cdncert/cert.pem"
 options:
   paged_rpm_inserts_limit: 100
+
 # metrics:
 #   path: "/metrics"
 #   port: 9000
+
+# Let you customize configurations for
+# the client services
+clients:
+  rbac_enabled: True
+  rbac_base_url: http://localhost:8800/api/rbac/v1
+  rbac_timeout: 30
+
+# Configuration for the mocks
+mocks:
+  # Update this with your org_id value
+  my_org_id: "12345"
+  rbac:
+    # Update with your account number for admin
+    user_write: ["jdoe@example.com"]
+    # Update with yout account number for viewer
+    user_read: ["tdoe@example.com"]
```

Comment:
make it simpler :) 
```
user_write: ["write"]
user_read: ["read]
```

I still think i'd prefer to allow write by all unless explicitly denied, so you'd have just:
```
user_denied: ["baduser"]
user_read:["reader"]
```

That way user opt-in to testing the rbac when they want, rather than be restricted to one or two users all the time.  i can see an argument that since this is an rbac mocking service, maybe it shouldn't default to open, but it is just a mocked service, never meant to be really used.

Reply:
I have updated with the below:

```raw
user_read_write: ["user1","user2"]
user_read: ["user3"]
```

that match with the permissions returned.

to don't use the rbac check during developing other functionalities, it can be disabled by:

```yaml
clients:
  rbac_enabled: False
```

so the configuration for the allowed read/write and read is kept, and we can use it when we need, or disable when we are developing other features; by default it is enabled.

> I will update `configs.yaml.example` file to add comments about the above.

Reply:
as `configs/config.yaml` is used only for local development, I have set `rbac_enabled: False` there, so if a new configuration is generated from it, by default the rbac will be disabled for the local environment.

I have added comments to clarify what happens when the value is True and False.